kubed-core/src/main/kotlin/kubed/scale/LinearScale.kt

```diff
@@ -29,7 +34,7 @@ open class LinearScale<R>(interpolate: (R, R) -> (Double) -> R,
         val stop: Double = domain.last()
         var step = tickStep(start, stop, count)
 
-        if(step > 0) {
+        if (step > 0) {
```

Comment:
A minor thing, but I noticed some formatting that I found unusual, like the one above, which is applied consistently, so it looks like it's intentional.
Perhaps you might consider https://kotlinlang.org/docs/reference/coding-conventions.html ?

```kotlin
if (condition) {
    // body
} else {
    // else part
}
```

Reply:
Yeah it’s intentional. I like many of the Kotlin guidelines but a space before ( (when the “(“ is required) and else on the same line as the ending { aren’t among them :-P

Reply:
Fair enough :)